cg/apps/slurm/slurm_api.py

```diff
@@ -0,0 +1,66 @@
+"""Class to create sbatch files and communicate with slurm"""
+import logging
+from pathlib import Path
+from typing import List, Optional
+
+from cg.apps.slurm.sbatch import SBATCH_BODY_TEMPLATE, SBATCH_HEADER_TEMPLATE
+from cg.models.slurm.sbatch import Sbatch
+from cg.utils import Process
+
+LOG = logging.getLogger(__name__)
+
+
+class SlurmAPI:
+    def __init__(self):
+        self.process: Process = Process("sbatch")
+        self.dry_run: bool = False
+
+    def set_dry_run(self, dry_run: bool) -> None:
+        LOG.info("Set dry run to %s", dry_run)
+        self.dry_run = dry_run
+
+    @staticmethod
+    def generate_sbatch(sbatch_parameters: Sbatch) -> str:
```

Comment:
generate_sbatch_script?

Reply:
If you like, no problem

Reply:
changed name to `generate_sbatch_content`